src/components/--Tag/Tag.stories.js

```diff
@@ -1,21 +1,23 @@
 import React from "react";
-
+import { colorScheme } from "../tokens";
 import Tag from "./Tag";
 
 export default {
   title: "Tag",
   component: Tag,
-  args: {
-    name: "tag name",
-  },
 };
 
-const Template = (args) => {
+export const Basic = (args) => {
   return <Tag {...args} />;
 };
 
-export const Default = Template.bind({});
-export const Yellow = Template.bind({});
-Yellow.args = {
-  color: "#ffcc00",
-};
+export const All = () => (
+  <>
+    {Object.entries(colorScheme).map(
+      ([colorName, color]) =>
+        colorName !== "light" && ( // white color is not visible
+          <Tag key={colorName} name={colorName} color={color} />
+        )
```

Comment:
Just for display for the moment. If we decide to color tags differently, we could probably set categoryColors in tokens to store only necessary colors.